modules/network/nsgs.tf

```diff
@@ -366,14 +364,25 @@ resource "oci_core_network_security_group_security_rule" "pods_egress" {
   count = var.cni_type =="npn" ? length(local.pods_egress) : 0

```

Comment:
Can you please add the same condition for the creation of the pod nsg?

Reply:
Done.